cmd/kubelet/app/server.go

```diff
@@ -57,8 +60,6 @@ import (
 	"k8s.io/kubernetes/pkg/capabilities"
 	"k8s.io/kubernetes/pkg/client/chaosclient"
 	"k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
```

Comment:
No, keeping it small is fine. We have tons of those renames to do. These few do not make a big difference.